docs/dev-run.js

```diff
@@ -64,7 +64,7 @@ portfinder.getPorts(2, {}, (portFinderErr, [docsPort, webpackPort]) => {
     process.exit(1);
   }
 
-  runCmd('webpack-dev-server', `nodemon --watch webpack --watch webpack.docs.js --exec webpack-dev-server -- --config webpack.docs.js --color --port ${webpackPort} --debug --hot --host ${ip.address()}`);
+  runCmd('webpack-dev-server', `nodemon --watch webpack --watch webpack.docs.js --exec webpack-dev-server -- --config webpack.docs.js --color --port ${webpackPort} --debug --hot --inline --host ${ip.address()}`);
```

Comment:
I'm not in this code much anymore so I'll leave that decision to you guys. My use case for this was to launch the docs on my mac, then load the docs site on my Windows VM in Internet Explorer. Since we all know how awesome Internet Explorer can be it may be beneficial to leave this in here so you guys can debug in that browser without hosting your dev environment on Windows. It's entirely up to you guys.